packages/playground/src/components/account/account-deposit/account-deposit.tsx

```diff
@@ -32,8 +32,8 @@ export class AccountDeposit {
     );
   }
 
-  async formSubmissionHandler(e) {
-    this.amountDeposited = ethers.utils.parseEther(e.target.value);
+  async formSubmitionHandler(e) {
```

Comment:
`submission` is actually the correct spelling

Reply:
lol, someone corrected this in master :P

Reply:
I'll fix it in another PR so I don't trigger another CI job here.